packages/mui-material/src/SvgIcon/SvgIcon.js

```diff
@@ -40,21 +40,21 @@ const SvgIconRoot = styled('svg', {
   display: 'inline-block',
   fill: 'currentColor',
   flexShrink: 0,
-  transition: theme.transitions.create('fill', {
-    duration: theme.transitions.duration.shorter,
+  transition: theme.transitions?.create?.('fill', {
+    duration: theme.transitions?.duration?.shorter,
   }),
   fontSize: {
     inherit: 'inherit',
-    small: theme.typography.pxToRem(20),
-    medium: theme.typography.pxToRem(24),
-    large: theme.typography.pxToRem(35),
+    small: theme.typography?.pxToRem?.(20),
+    medium: theme.typography?.pxToRem?.(24),
+    large: theme.typography?.pxToRem?.(35),
```

Comment:
Shouldn't we guard against cases where theme.typograhy is undefined? Perhaps a fallback value?

Reply:
Sure, I can add to it.